TypeScript.py

```diff
@@ -792,6 +795,29 @@ def on_close(self, view):
                     cli.service.close(view.file_name())
         logger.view_debug(view, "exit on_close")
 
+    def handlePulse(self):
```

Comment:
There is no way to detect scrolling in the Sublime API, so we need a periodic timer to check whether the view has been scrolled and to update placement of the error status icon.